src/input/input_abstract.rs

```diff
@@ -96,17 +96,17 @@ pub trait Input<'a>: fmt::Debug + ToPyObject {
         self.strict_int()
     }
 
-    fn validate_float(&self, strict: bool) -> ValResult<f64> {
+    fn validate_float(&self, strict: bool, only_finite: bool) -> ValResult<f64> {
```

Comment:
I think we should do this check in `src/validators/float.rs`, since:
1. The check should be the same in in the all 3 input types
2. In theory we might add more input types in future and these checks would need to be duplicated to them

Reply:
You're absolutely right. This was a terrible mistake

Reply:
:confused: I think at most it was a "mistake", nothing terrible is happening here :-)